docs/Data_Portal/Users_Guide/oncomatrix.md

```diff
@@ -180,164 +174,170 @@ The row containing TP53 cases now moves back up in position 1 above ATRX.
 
-Note that these sorting options correspond to SSM only. For CNV, user has to show CNV data using the CNV button for sorting options to become viewable. This step is explained in the 'CNV' section below.
+These sections are described below.
 
-__Maximum #cases__
+#### Maximum #cases
 
-There is a default number of samples that are shown in the matrix chart. Users can choose to increase or decrease the number of samples. This allows the chart to re-render and display the number of columns based on the user's selection. Figure below shows increased cases to 10000. Please note that any high arbitrary number can be selected but the chart will only show the maximum cases that GDC has.
+There is a default number of samples that are shown in the matrix chart. Users can choose to increase or decrease the number of samples. This allows the chart to re-render and display the number of columns based on the user`s selection. Figure below shows increased cases to 10000. Please note that any high arbitrary number can be selected but the chart will only show the maximum cases that GDC has.
 
-[![](./images/oncomatrix/max-cases.png)](./images/oncomatrix/max-cases.png 'Click to see the full image.')
+[![Cases menu](./images/oncomatrix/20.1-by_consequence.png)](./images/oncomatrix/20.1-by_consequence.png 'Click to see the full image.')
 
 The chart will reload with new cases added.
 
-__Group cases by__
+#### Case Label Character Limit
+
+This option allows users to increase or decrease the length of the case label. The default number is 32 characters. The chart will reload with new cases added.
+
+#### Group cases by
 
 This option allows users to group cases by different variables from the GDC dictionary. Click on the `+` icon shown in blue to display different variables such as demographics, diagnoses, Exposures etc. Users may also search for a variable from the search bar provided in the menu as shown by `Search Variables`.
 
 [![Group cases by another variable](./images/oncomatrix/group-cases-by.png)](./images/oncomatrix/group-cases-by.png 'Click to see the full image.')
 
-Click 'Age at diagnosis' from the options. The matrix reloads to show the following view.
+Search for the term ‘Age at diagnosis’ and Click the term. The matrix reloads to show the following view.
 
 [![Select a grouping variable](./images/oncomatrix/22-group-age-at-diag.png)](./images/oncomatrix/22-group-age-at-diag.png 'Click to see the full image.')
 
 As shown above, labels for different age groups show up vertically and all cases get distributed with a clearcut separation according to the age bins.
 
-Click on the 'Age at diagnosis' (blue pill, as shown). This opens a short menu with action items. Click on the first item 'Edit' as shown.
+Click on the ‘Age at diagnosis’ (blue pill, as shown). This opens a short menu with action items. Click on the first item ‘Edit’ as shown.
```

Comment:
Is the replacement of straight quotation marks with curved quotation marks intentional? Sometimes our doc site scripts cannot render curved quotation marks correctly and we have had to be correct these back before. See this line and lines below.

Reply:
Thanks, I'll revert to double-quotes.

Reply:
@bullenc Actually, before I'd like to verify first: by common standard, back ticks will format the enclosed text as code snippets, with a grayed out text background. I'm hesitant to remove that styling, unless it really breaks the build.

Reply:
@siosonel we can move forward and I will let you know if it breaks the build. My understanding is that the code back ticks tend to point in the other direction (``) whereas these seem to be curved quotations (‘’) vs. the original straight quotations ('').

Reply:
ah replied late, no worries, thanks!